$MirOS: ports/net/djbdns/patches/patch-src_log_h,v 1.1 2005/12/14 23:28:35 tg Exp $
--- src/log.h.orig	Sun Feb 11 21:11:23 2001
+++ src/log.h	Sat Mar 26 15:33:25 2016
@@ -18,11 +18,13 @@ extern void log_cachednxdomain(const cha
 extern void log_cachedns(const char *,const char *);
 
 extern void log_tx(const char *,const char *,const char *,const char *,unsigned int);
+extern void log_tx_piggyback(const char *,const char *,const char *);
 
 extern void log_nxdomain(const char *,const char *,unsigned int);
 extern void log_nodata(const char *,const char *,const char *,unsigned int);
 extern void log_servfail(const char *);
 extern void log_lame(const char *,const char *,const char *);
+extern void log_ignore_referral(const char *,const char *,const char *);
 
 extern void log_rr(const char *,const char *,const char *,const char *,unsigned int,unsigned int);
 extern void log_rrns(const char *,const char *,const char *,unsigned int);
